### ISSUE TITLE
[ic-stepper] colour contrast fails on page header for step subtitle text

### ISSUE THREAD

**Issue by GCHQ-Developer-530** [mi6/ic-ui-kit#1907]
## Summary of the bug 

Title: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (ic-step[step-title="Warm milk"],.step-subtitle)
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 4.25 (foreground color: #6c7580, background color: #f4f4f5, font size: 9.0pt (12px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Chrome version 125.0.0.0

====

## 🪜 How to reproduce 
Tell us the steps to reproduce the problem: 
1. Go to any page header example with stepper
2. Run accessibility fast pass
4. See the error 

## 🧐 Expected behaviour 
It should pass colour contrast.

## Additional info 
It looks like the solution is to update the step subtitle text to be #666f79, but does this just need updating when it's on a page header or should we update it for all steppers?

**Comment by mi6-577** [comment]
Amended step-title-status-text-inactive token from text/tertiary to text/secondary #41464D (LM) / #E1E3E5 (DM) which now meets WCAG guidelines with 8:65:1 on the page header background. It also falls in line with the text colours used in compact stepper.

This change has been made on the v3 Figma design file, ready for dev

**Comment by MI6-255** [comment]
Might be done by @MI6-286 or @ad9242

**Comment by MI6-286** [comment]
This has been resolved now that the colours have been updated as per the comment above. The disabled stepper text colour still fails colour ratio checks - a separate ticket has been raised for this #3422.
